src/lib/util/FormatMessage.ts

```diff
@@ -32,26 +32,38 @@ export const formatMessage = async (message: Message) => {
         // For one liners
         if (language.trim() && !languageMappings[language]) {
           unformattableCodeBlockCounter++;
+          const commented = commentify(match, firstLanguageKey);
+          contentWithoutCode = contentWithoutCode.replace(match, '\n');
+          // Replace this line with it's commented version 
+          prettiedPastecordCode = prettiedPastecordCode.replace(
+            match,
+            commented
+          );
+          // returning match itself means nothing will be changed
           return match;
         }
         // For empty codeblocks, remove the content
         unformattableCodeBlockCounter++;
+        // Returning '' will completely remove the empty code block
         return '';
       }
-      // If firstLanguage is undefined and language is a valid languagekey, it sets the value
-      if (!firstLanguageKey && languageNameMappings[language])
```

Comment:
as mentioned above, now we'll assign it on declaring firstLanguageKey instead of going through all of it